src/main/java/org/zapply/product/global/security/threads/ThreadsProfile.java

```diff
@@ -0,0 +1,11 @@
+package org.zapply.product.global.security.threads;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record ThreadsProfile(
+        String id,
+        String username,
+        String name,
+        @JsonProperty("threads_profile_picture_url") String profilePictureUrl,
+        @JsonProperty("threads_biography") String biography
```

Comment:
```suggestion
public record ThreadsProfile(
        String id,
        
        String username,
        
        String name,
        
        @JsonProperty("threads_profile_picture_url") 
        String profilePictureUrl,
        
        @JsonProperty("threads_biography") 
        String biography
```

record 타입 작성 방식을 통일하면 좀 더 일관성있을 것 같은데, 선호하는 작성 방식이 있으신가요? 저는 이런식으로 한 줄에 하나씩 작성하는 걸 선호하는 편이에요!

Reply:
가독성을 위해서 통일하면 좋을 것 같네요! 수정해두겠습니다~